File: server/src/main/java/gg/mineral/server/entity/living/Zombie.java
package gg.mineral.server.entity.living;

import gg.mineral.server.entity.LivingImpl;
import gg.mineral.server.world.WorldImpl;

public class Zombie extends LivingImpl {

    public Zombie(int id, WorldImpl world) {
        super(id, world);
    }

    @Override
    public void tickAsync() {
    }

}

File: server/src/main/kotlin/gg/mineral/server/entity/living/Zombie.kt
package gg.mineral.server.entity.living

import gg.mineral.server.entity.LivingImpl
import gg.mineral.server.snapshot.AsyncServerSnapshotImpl
import gg.mineral.server.world.WorldImpl

class Zombie(id: Int, serverSnapshot: AsyncServerSnapshotImpl, world: WorldImpl) : LivingImpl(id, serverSnapshot, world)
